### PR TITLE
Adds Damn Daniel Shoes

### PR DESCRIPTION
**Pull request by chesse20**
[Changelogs]: Adds shoes that when you point at them your character says "damn daniel" and they are admin spawnable

:cl: optional name here
add: Adds Dam Daniel Shoes
/:cl:

[why]: Funny and Cool

### CODE REVIEW

**Comment by chesse20** [comment]
░░░░░░░░░░░░▄▄                     
 ░░░░░░░░░░░█░░█                  
 ░░░░░░░░░░░█░░█                  
 ░░░░░░░░░░█░░░█                  
 ░░░░░░░░░█░░░░█                  
 ███████▄▄█░░░░░██████▄ 
 ▓▓▓▓▓▓█░░░░░░░░░░░░░░█ 
 ▓▓▓▓▓▓█░░░░ /tg/, ░░░░░░█ 
 ▓▓▓▓▓▓█░░░░Add This░░░░█  
 ▓▓▓▓▓▓█░░░░Please!░░░░░█ 
 ▓▓▓▓▓▓█░░░░░░░░░░░░░░█ 
 ▓▓▓▓▓▓█████░░░░░░░░░█   
 ██████▀░░░░▀▀██████▀

**Comment by nichlas0010** [comment]
🤔

**Comment by Time-Green** [comment]
Add a point reaction method, instead of a snowflake check

Otherwise this seems high quality and will be merged

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -319,6 +319,18 @@
 
 	new /obj/effect/temp_visual/point(A,invisibility)
 
+	var/obj/item/clothing/shoes/sneakers/white/damndaniel/damn_daniel
+
+	if(istype(A, /obj/item/clothing/shoes/sneakers/white/damndaniel/))
+		damn_daniel = A
+	else
+		for(var/obj/item/clothing/shoes/sneakers/white/damndaniel/DamnDanielShoes in A.contents)
+			damn_daniel = DamnDanielShoes
```
Comment (optimumtact) [inline code/modules/mob/mob.dm]: lmao wtf is this shit code lol

**Comment by JStheguy** [comment]
This PR gave me space cancer.

**Comment by 81Denton** [comment]
That's a two year old low quality meem
You should feel bad

**Comment by chesse20** [comment]
i fixed it! :yum:

File: code/modules/clothing/shoes/colour.dm [hunk 2]
```diff
@@ -75,6 +75,10 @@
 	permeability_coefficient = 0.01
 	item_color = "white"
 
+/obj/item/clothing/shoes/sneakers/white/damndaniel/pointed_at(var/mob/user)
+	spawn(10)
```
Comment (optimumtact) [inline code/modules/clothing/shoes/colour.dm]: use add timer you dumbo

File: code/modules/mob/mob.dm [hunk 3]
```diff
@@ -319,6 +321,9 @@
 
 	new /obj/effect/temp_visual/point(A,invisibility)
 
+	for(var/atom/on_tile in A.contents + A)
```
Comment (optimumtact) [inline code/modules/mob/mob.dm]: lmao no, dumbo, use signal you are not very smart at all

File: code/modules/mob/mob.dm [hunk 4]
```diff
@@ -300,6 +300,8 @@
 	face_atom(A)
 	A.examine(src)
 
+/atom/proc/pointed_at(var/mob/user)
```
Comment (optimumtact) [inline code/modules/mob/mob.dm]: absolute state of hippie coder lol

Comment (praisenarsie) [reply code/modules/mob/mob.dm]: he's a hippie coder? i thought he was someone from para or citadel.

**Comment by ike709** [comment]
dead meme

**Comment by Borisvanmemes** [comment]
Yeah this really is a stale and dead meme

**Comment by Cyberboss** [comment]
This PR gave me cancer

**Pull request closed without merging**